linter.py

```diff
@@ -1,5 +1,21 @@
+import os
 from SublimeLinter.lint import NodeLinter
 
+# TODO Proper export these in SL core
```

Comment:
Can you open an issue on SublimeLinter about properly exporting these?

Reply:
https://github.com/SublimeLinter/SublimeLinter/issues/1713